snapcraft.yaml

```diff
@@ -23,11 +20,16 @@ parts:
       - voluptuous
     prime:
       - bin/ubuntu-image
+      - bin/dd
+      - bin/cp
```

Comment:
What is the reason for including coreutils (dd/cp) here?  I don't see any reason to pull them into the snap when we should be able to reasonably rely on them being present on the underlying system.  All the other dependencies are things that we can't guarantee will be present on the host, but coreutils is ubiquitous (it's Essential: yes, so you'll never find a Debian or Ubuntu environment without it).